src/Datadog.Trace.ClrProfiler.Native/util.cpp

```diff
@@ -78,8 +78,8 @@ std::vector<WSTRING> GetEnvironmentValues(const WSTRING &name,
   return values;
 }
 
-std::vector<WSTRING> GetEnvironmentValues(const WSTRING &name) {
-  return GetEnvironmentValues(name, L';');
+std::vector<WSTRING> GetValues(const WSTRING &delimited_values) {
+  return GetValues(delimited_values, L';');
```

Comment:
Unrelated to this PR, but this should really default to `;` on Windows and `:` on Linux.

Reply:
I'm hesitant to change the default behavior right now in this PR, but this would make for a good follow-up PR